src/launch/launch_config.cpp

```diff
@@ -308,6 +314,7 @@ void LaunchConfig::parseNode(TiXmlElement* element, ParseContext ctx)
 	const char* coredumpsEnabled = element->Attribute("enable-coredumps");
 	const char* cwd = element->Attribute("cwd");
 	const char* clearParams = element->Attribute("clear_params");
+	const char* stopTimeout = element->Attribute("stop_timeout");
```

Comment:
Two points here:
 * roslaunch is not consistent with underscores and hyphens (`launch-prefix` vs. `clear_params`), but I'd lean towards hyphens - feels more like XML/HTML to me.
 * I'd actually prefer "rosmon-stop-timeout", since we don't control the roslaunch XML spec - they might decide to add something like this later on with a slightly different meaning. I know `enable-coredumps` doesn't have the prefix, but we don't have to repeat the same mistake ;-)

Reply:
OK, I'll fix this on Monday morning